wikiclass/feature_lists/enwiki.py

```diff
@@ -60,6 +82,7 @@
     main_article_templates / max(wikitext.revision.content_chars, 1),
     (english.stemmed.revision.stem_chars /
      max(wikitext.revision.content_chars, 1)),
+    paragraphs_without_refs_total_length,
```

Comment:
One thing, this without scaling can cause abnormalities in the result, you need to add "log" to counting features, look at https://github.com/wiki-ai/wikiclass/blob/9e7660d6a2054119662de887e79456f44d9adf98/wikiclass/feature_lists/wikibase.py for example

Reply:
Done!